client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/eligibility-warnings-modal.tsx

```diff
@@ -0,0 +1,37 @@
+import { Dialog } from '@automattic/components';
+import EligibilityWarnings from 'calypso/blocks/eligibility-warnings';
+import type { SiteDetails } from '@automattic/data-stores';
+import './eligibility-warnings-modal.scss';
+
+export const EligibilityWarningsModal = ( {
+	site,
+	isMarketplace,
+	isOpen,
+	handleClose,
+	handleContinue,
+}: {
+	site?: SiteDetails;
+	isMarketplace?: boolean;
+	isOpen: boolean;
+	handleClose: () => void;
+	handleContinue: () => void;
+} ) => {
+	return (
+		<Dialog
+			additionalClassNames="eligibility-warnings-modal__dialog-content"
+			isVisible={ isOpen }
+			onClose={ handleClose }
+			showCloseIcon={ true }
+		>
+			<EligibilityWarnings
+				siteId={ site?.ID }
+				currentContext="plugin-details"
+				standaloneProceed
+				isMarketplace={ isMarketplace }
+				isMarketplaceException={ isMarketplace }
+				onProceed={ handleContinue }
+				backUrl="#"
```

Comment:
Question: Maybe we don't need this property? And if its being used somewhere we might have to change this, right?

Reply:
Good point. Removed in this commit https://github.com/Automattic/wp-calypso/commit/07b49d2ed47b724e92eba97db96730d7046edc14 and reason explained in https://github.com/Automattic/wp-calypso/pull/81412#discussion_r1327367858

Reply:
Awesome!